gbpservice/nfp/test/unit/configurator/agent/lb_test_driver.py

```diff
@@ -1,360 +1,18 @@
-import requests
```

Comment:
Please give other file name. lb_test_driver.py is generic one but thi is HaproxyOnVmDriver.